README.md

```diff
@@ -66,20 +64,19 @@ module "vault_secretsync" {
   associate_secrets = {
     foo = {
       mount       = "mount_foo"
-      secret_name = "foo_secret"
+      secret_name = ["foo_secret"]
     }
     hello = {
       mount       = "mount_hello"
-      secret_name = "hello_secret"
+      secret_name = ["hello_secret"]
     }
   }
 
   delete_all_secret_associations = true
-  delete_sync_destination	     = true
 }
 ```
 
-Remove vault secret sync destination:
+Remove vault secret sync destination (all secret associations must be removed before this can be done):
```

Comment:
```suggestion
Remove vault secret sync destination by adding `delete_sync_destination = true` (NOTE: all secret associations must be removed before this can be done):
```